apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -146,39 +146,40 @@ function handlePostLocked( calypsoPort ) {
 	const unsubscribe = subscribe( () => {
 		const isLocked = select( 'core/editor' ).isPostLocked();
 		const isLockTakeover = select( 'core/editor' ).isPostLockTakeover();
-		const lockedDialogButtons = document.querySelectorAll(
-			'div.editor-post-locked-modal__buttons > a'
-		);
-
-		const isPostTakeoverDialog = isLocked && ! isLockTakeover && lockedDialogButtons.length === 3;
 
-		if ( isPostTakeoverDialog ) {
-			//signal the parent frame to navigate to All Posts
-			lockedDialogButtons[ 0 ].addEventListener(
-				'click',
-				( event ) => {
-					event.preventDefault();
-					calypsoPort.postMessage( { action: 'goToAllPosts' } );
-				},
-				false
+		if ( isLocked && ! isLockTakeover ) {
+			const lockedDialogButtons = document.querySelectorAll(
+				'div.editor-post-locked-modal__buttons > a'
 			);
 
-			//overrides the all posts link just in case the user treats the link... as a link.
-			if ( calypsoifyGutenberg && calypsoifyGutenberg.closeUrl ) {
-				lockedDialogButtons[ 0 ].setAttribute( 'target', '_parent' );
-				lockedDialogButtons[ 0 ].setAttribute( 'href', calypsoifyGutenberg.closeUrl );
-			}
+			if ( lockedDialogButtons.length === 3 ) {
```

Comment:
I'm confused, my dialog just has one button:

```
document.querySelectorAll('div.editor-post-locked-modal__buttons > a');
NodeList [ a.components-button.is-primary ]
0: <a class="components-button is-primary" href="edit.php?post_type=post">
length: 1
```

Reply:
Looking upstream, I think there an only ever be two buttons for the modal: https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/post-locked-modal/index.js

Reply:
It used to be three buttons, but changed to two with https://github.com/WordPress/gutenberg/pull/37852, this probably needs updating. I still can't trigger it so maybe something else is outdated....I'll open an issue

Reply:
https://github.com/Automattic/wp-calypso/issues/83426